### PR TITLE
Add new API version for action group resource provider

### PR DESCRIPTION
**Pull request by kikicandy11**
Add new field "userUnifiedSchema" for action

### CODE REVIEW

File: schemas/2019-03-01/Microsoft.Insights.ActionGroup.tests.json [hunk 1]
```diff
@@ -0,0 +1,307 @@
+{
+  "tests": [
```
Comment (TianoMS) [inline schemas/2019-03-01/Microsoft.Insights.ActionGroup.tests.json]: >tests [](start = 3, length = 5)

please put the test file in the /tests folder.

**Comment by Tiano2017** [comment]
@kikicandy11 one comment. please take a look.

**Comment by kikicandy11** [comment]
@Tiano2017 fixed, thx

**Comment by kikicandy11** [comment]
Hi, @Tiano2017 could you take a look at the PR

**Pull request merged**